docs/src/development/variables/set-variables.md

```diff
@@ -0,0 +1,158 @@
+| Runtime   | `--visible-runtime` | `true`  | Whether the variable is available at runtime. |
+| Build     | `--visible-build`   | `true`  | Whether the variable is available at build time. |
+
+So if you want the `foo` variable to be visible at build time but hidden during runtime, you can set it like this:
+
+```bash
+platform variable:create --level project --name foo --value bar --visible-build true --visible-runtime false
+```
+
+You can also change the variable options after you create them (except for sensitive values, which can't be set to non-sensitive). For example, to make the `foo` variable visible at runtime and hidden during the build, run this command:
+
+```bash
+platform variable:update foo --visible-build false --visible-runtime true
+```
+
+Note that any changes to project variables require you to deploy your environments to have effect.
+
+## Create environment-specific variables
+
+Set variables for specific environments using [the management console](../../administration/web/configure-environment.md#variables) or the CLI.
+Variables can be inherited or overridden from parent environments and project variables.
+See [more on overriding values](./_index.md#overrides)
+
+For example, to create the environment variable `foo` with the value `bar` on the current environment, run:
+
+```bash
+platform variable:create --level environment --name foo --value bar  --visible-build true --visible-runtime false
+```
+
+To specify the environment for the variable, use the `-e` flag to specify its name.
+
+When naming variables, be sure to take [variable prefixes](./_index.md#variable-prefixes) into account.
+
+### Environment variable options
+
+Environment variables share all of the [options available for project variables](./_index.md#variable-options), with the exception that visibility in the build and runtime can be set only with the CLI (not in the console). Environment variables have one additional option:
+
+| Option      | CLI flag        | Default | Description |
+| ----------- | --------------- | ------- | ----------- |
+| Inheritable | `--inheritable` | `true`  | Whether the variable is inherited by child environments. |
+
+This option is useful for  setting production-only values such as credentials. For example, to set a PayPal secret value for only the `main` branch and have it not be readable elsewhere, run:
+
+```bash
+platform variable:create -e main --name paypal_id --inheritable false --sensitive true
+```
+
+Other environments don't inherit it and get either a project variable of the same name if it exists or no value at all.
+
+Note that changing an environment variable causes that environment to be redeployed so the new value is available.
+However, child environments are *not* redeployed.
+To make the new value accessible to those environments, redeploy them manually.
```

Comment:
```suggestion
To make the new value accessible to those environments, redeploy them either via:
- [the console](../../administration/web/_index.md),
- [the cli](/cli.md) `platform redeploy -e <name of the child environment>`.
```

Reply:
We also already have instructions for this: https://docs.platform.sh/development/troubleshoot.html#force-a-redeploy
So I'd just link there.

Reply:
Edited the suggestion to link to the page you provided ;) thanks!